packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -351,6 +359,7 @@ export function doStandardCalcs(field: Field, ignoreNulls: boolean, nullAsZero:
 
   if (isNumber(calcs.firstNotNull) && isNumber(calcs.lastNotNull)) {
     calcs.diff = calcs.lastNotNull - calcs.firstNotNull;
+    calcs.diffperc = calcs.diff / calcs.firstNotNull;
```

Comment:
Maybe a test for this calculation would be a good idea, just to make sure no one breaks this in the future. You calc add it in the fieldReducer.test.ts